src/styles/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+  width: 96px;
+  height: 16px;
+  margin: 0;
```

Comment:
7'th point in checklist

Reply:
7. [BEM] - Make sure to follow BEM naming convention ??